src/MIDebugEngine/AD7.Impl/AD7Property.cs

```diff
@@ -111,6 +115,15 @@ public DEBUG_PROPERTY_INFO ConstructDebugPropertyInfo(enum_DEBUGPROP_INFO_FLAGS
                     propertyInfo.dwAttrib |= enum_DBG_ATTRIB_FLAGS.DBG_ATTRIB_VALUE_RAW_STRING;
                 }
                 propertyInfo.dwAttrib |= variable.Access;
+
+                if (_uiVisualizers != null && _uiVisualizers.Any())
+                {
+                    propertyInfo.dwAttrib |= enum_DBG_ATTRIB_FLAGS.DBG_ATTRIB_VALUE_CUSTOM_VIEWER;
+                    if (_uiVisualizers.Count() > 1)
```

Comment:
>.Count() [](http://example.com/codeflow?start=38&length=8)

nit: this will wind up doing too passes over the enumerable. Should we just turn the enumerable into an array in the constructor?